x/oracle/types/error.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var (
+	ErrOwasmCompilation         = sdkerrors.Register(ModuleName, 1, "owasm compilation failed")
+	ErrBadWasmExecution         = sdkerrors.Register(ModuleName, 2, "bad wasm execution")
+	ErrDataSourceNotFound       = sdkerrors.Register(ModuleName, 3, "data source not found")
+	ErrOracleScriptNotFound     = sdkerrors.Register(ModuleName, 4, "oracle script not found")
+	ErrRequestNotFound          = sdkerrors.Register(ModuleName, 5, "request not found")
+	ErrRawRequestNotFound       = sdkerrors.Register(ModuleName, 6, "raw request not found")
+	ErrReporterNotFound         = sdkerrors.Register(ModuleName, 7, "reporter not found")
+	ErrResultNotFound           = sdkerrors.Register(ModuleName, 8, "result not found")
+	ErrReporterAlreadyExists    = sdkerrors.Register(ModuleName, 9, "reporter already exists")
+	ErrValidatorNotRequested    = sdkerrors.Register(ModuleName, 10, "validator not requested")
+	ErrValidatorAlreadyReported = sdkerrors.Register(ModuleName, 11, "validator already reported")
+	ErrInvalidReportSize        = sdkerrors.Register(ModuleName, 12, "invalid report size")
+	ErrReporterNotAuthorized    = sdkerrors.Register(ModuleName, 13, "reporter not authorized")
+	ErrEditorNotAuthorized      = sdkerrors.Register(ModuleName, 14, "editor not authorized")
```

Comment:
missing the 15 or is It ok?

Reply:
@bragaz These files are automatically downloaded from the Band Protocol repository, so I think it's fine. We only use part of them anyway inside our codebase